pages/common/lab.md

```diff
@@ -0,0 +1,19 @@
+# lab
+
+> A CLI for Gitlab.
+
+- Create a merge request and specify remote and target branch:
+
+`lab mr create {{remote}} {{target_branch}}`
+
+- Create a pull request on a github repo:
```

Comment:
Shouldn't it be "gitlab" repo ? And we should rename it to "merge request" in keeping parity with gitlab terminology.